api/views.py

```diff
@@ -4,23 +4,43 @@
 from django.contrib.auth.models import User
 from django.views.decorators.csrf import csrf_exempt
 from django.http import JsonResponse, HttpResponseBadRequest
+from django.contrib.auth import authenticate, login, logout
+from django.contrib.auth.decorators import login_required
+from django.views.decorators.http import require_POST, require_GET
 
 @csrf_exempt
 def create_user(request):
     if request.method == 'POST':
-        data = json.loads(request.body)
+        try:
```

Comment:
What does 'try:' do here?

Reply:
its like error handling in rails
it looks like JS tho
try:
do this code (if it errors)
exception(an error catch)
return this error json